File: 1065.cpp
#include <iostream>
using namespace std;

int main() {
	int N, cnt = 0;
	int a, b, c;

	cin >> N;

	for (int i = 1; i < N + 1; i++) {
		if (i < 100)cnt++;
		else {
			a = i / 100;
			c = i % 10;
			b = (i % 100) / 10;
			if (b-a == c-b)cnt++;
		}
	}
	
	cout << cnt << '\n';

	return 0;
}

File: 1065.py
N = int(input())
cnt = 0
a = b = c = 0

for i in range(1,N+1) :
    if i < 100 : 
        cnt += 1
        continue
    a = i%10
    b = (i//10)%10
    c = i//100
    if (a-b)==(b-c) :
        cnt += 1

print (cnt)